projects/assets-library/src/icons/icon-library.module.ts

```diff
@@ -43,6 +43,7 @@ const iconsRootPath = 'assets/icons';
         { key: IconType.Edge, url: `${iconsRootPath}/edge.svg` },
         { key: IconType.ExpandAll, url: `${iconsRootPath}/expand-all.svg` },
         { key: IconType.Expanded, url: `${iconsRootPath}/minus-circle.svg` },
+        { key: IconType.ExpandedReverse, url: `${iconsRootPath}/minus-circle-reverse.svg` },
```

Comment:
Is this meant to take the place of the current solid minus-circle on the line above? If yes, would it make sense to just update that svg rather than introduce the parallel one? If not, I'm confused when we'd use one vs the other

Reply:
It is a new icon that is the same as the Minus Circle Icon but reversed.